addons/account/tests/test_transfer_wizard.py

```diff
@@ -190,6 +190,66 @@ def setUpClass(cls, chart_template_ref=None):
         })
         cls.move_2.action_post()
 
+    def test_transfer_default_tax(self):
+        """ Make sure default taxes on accounts are not computed on transfer moves
+        """
+        account_with_tax = self.env['account.account'].create({
+            'name': 'Auto Taxed',
+            'code': 'autotaxed',
+            'account_type': 'expense',
+            'tax_ids': [Command.link(self.company_data['default_tax_purchase'].id)],
+        })
+        expense_accrual_account = self.env['account.account'].create({
+            'name': 'Accrual Expense Account',
+            'code': '234567',
+            'account_type': 'expense',
+            'reconcile': True,
+        })
+        move_with_tax = self.env['account.move'].create({
+            'journal_id': self.journal.id,
+            'line_ids': [
+                Command.create({
+                    'account_id': account_with_tax.id,
+                    'balance': 400,
+                }),
+                Command.create({
+                    'account_id': self.payable_account.id,
+                    'balance': -460,
+                }),
+            ]
+        })
+        move_with_tax.action_post()
+
+        self.assertRecordValues(move_with_tax.line_ids, [
+            {'balance': 400, 'account_id': account_with_tax.id},
+            {'balance': -460, 'account_id': self.payable_account.id},
+            {'balance': 60, 'account_id': self.company_data['default_account_tax_purchase'].id},
+        ])
+
+        # Open the transfer wizard
+        wizard = self.env['account.automatic.entry.wizard'] \
+            .with_context(active_model='account.move.line', active_ids=move_with_tax.line_ids[0].ids) \
+            .create({
+                'action': 'change_period',
+                'date': '2019-05-01',
+                'journal_id': self.company_data['default_journal_misc'].id,
+                'expense_accrual_account': expense_accrual_account.id,
+            })
```

Comment:
I think this is a good candidate for the `Form` context manager (like it's already done in the other tests) :blush:

Reply:
I made the change, but what is the exact benefit of using `Form` here?